android/engine/src/main/java/org/smartregister/fhircore/engine/rulesengine/services/DateService.kt

```diff
@@ -37,4 +37,58 @@ object DateService {
         )
     }
   }
+
+  @JvmOverloads
+  fun addOrSubtractTimeUnitFromCurrentDate(
+    timeUnitCount: Int,
+    operation: String,
+    timeUnit: String = TimeUnit.YEAR.name,
```

Comment:
I think a better name for `Timeunit` would be `Period` or `Duration`

Reply:
I'd prefer TimeUnit. That's what it is called here `android.icu.util.TimeUnit` and here `java.util.concurrent.TimeUnit`